documentation/modules/creating-storage-mapping.adoc

```diff
@@ -28,6 +28,7 @@ You can create a storage mapping by using the {ocp} web console to map source di
 .. If your source provider is VMware, select a *Source datastore* and a *Target storage class*.
 .. If your source provider is {rhv-full}, select a *Source storage domain* and a *Target storage class*.
 .. If your source provider is {osp}, select a *Source volume type* and a *Target storage class*.
+.. If your storage provider is {virt}. select a *Source provider* and a *Target provider*.
```

Comment:
should be `Source storage class` and `Target storage class`, does it really refer to "provider" at the moment?

Reply:
![ocp_ocp_storage map](https://github.com/kubev2v/forklift-documentation/assets/60698649/9a4f3405-7ad1-458a-9d36-7b4715ee5944)
This is what I saw using OCP providers @qiyuann created for me -- but maybe i did something wrong? Xin, what are you seeing for this step?

Reply:
@RichardHoch two comments:

-  I suggest adding another step between step 3 to 4 - 

   After following step 3, the dialog looks as follows:

   ![Screenshot from 2023-08-16 21-23-17](https://github.com/kubev2v/forklift-documentation/assets/18169498/1674f7f9-8853-4b6e-8bee-58062012483d)

   There are 2 options for the user now:
    a. If he wants to create an **empty** mapping (i.e. declare the source and target providers but leave the specific storage mappings empty for future use) then jump to step 6 by clicking the **Create** button.

     b. If he wants to set the mapping(s) instead of leaving it empty, then in step 4 we need to add that the user should click the **Add** button to create the  first  storage mappings. 

   After clicking the **Add** button in (b), the dialog looks like this:

      ![Screenshot from 2023-08-16 21-31-50](https://github.com/kubev2v/forklift-documentation/assets/18169498/22ebfe88-a3cc-4d19-a823-3c95e8d682a3)

    **Please note** that this comment is not relevant specifically to OCP - OCP migration but to all providers and also seems relevant to the network mapping dialog (`step 4.3 Creating a network mapping`).

- Regarding step 4.d, it should be `Source storage class` and `Target storage class` as mentioned by Arik. 
  Currently there is a [bug](https://github.com/kubev2v/forklift-console-plugin/issues/691) since the `Source` label appears for the source instead of `Source storage class`. We'll fix it.

Reply:
> This is what I saw using OCP providers @qiyuann created for me -- but maybe i did something wrong? Xin, what are you seeing for this step?

The Source/Target storage class could appear when the `Add` button is clicked.